example/main.tf

```diff
@@ -1,13 +1,12 @@
 data "azurerm_subnet" "postgres" {
-  name                 = "aks"
+  name                 = "core-infra-subnet-0-${var.env}"
   resource_group_name  = "core-infra-${var.env}"
   virtual_network_name = "core-infra-vnet-${var.env}"
 }
 
 module "db" {
   providers = {
```

Comment:
this whole providers block should be removed

Reply:
Done - should have removed it